docs/federated/apollo-federation.md

```diff
@@ -9,9 +9,80 @@ feasible making it much harder to manage and leading to unnecessary bottlenecks.
 an API gateway and a number of smaller GraphQL services behind it alleviates some of those problems and allows teams to
 scale their graphs more easily.
 
-[Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/) is an architecture for 
-composing multiple GraphQL services into a single graph. Federated schemas rely on a number of custom directives to 
-instrument the behavior of the underlying graph and convey the relationships between different schema types. Each individual 
-GraphQL server generates a valid GraphQL schema and can be run independently. This is in contrast with traditional schema 
-stitching approach where relationships between individual services, i.e. linking configuration, is configured at the GraphQL 
+[Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/) is an architecture for
+composing multiple GraphQL services into a single graph. Federated schemas rely on a number of custom directives to
+instrument the behavior of the underlying graph and convey the relationships between different schema types. Each individual
+GraphQL server generates a valid GraphQL schema and can be run independently. This is in contrast with traditional schema
+stitching approach where relationships between individual services, i.e. linking configuration, is configured at the GraphQL
 Gateway level.
+
+## Install
+Using a JVM dependency manager, simply link `graphql-kotlin-federation` to your project.
+
+With Maven:
+
+```xml
+<dependency>
+  <groupId>com.expediagroup</groupId>
+  <artifactId>graphql-kotlin-federation</artifactId>
+  <version>${latestVersion}</version>
+</dependency>
+```
+
+With Gradle:
+
+```kotlin
+implementation("com.expediagroup", "graphql-kotlin-federation", latestVersion)
```

Comment:
within our build files we just use single string - we probably should follow same pattern? guess it doesn't matter

Reply:
I like the separated format to more clearly see the group, rather than looking for the `:` inside a string